### DISCUSSION TITLE
ACCOUNT SUSPENDED AND PURCHASED DEV++

### DISCUSSION THREAD

**Discussion by gandolfslayer** [forem/forem#21250] [Q&A]
I made my account and purchased Dev++ not even a day ago. It's been maybe 5hrs now and my account got suspended for some reason. On top of this, unknowingly I purchased Dev++ which I am unable to access due to the account suspension. I would very much so appreciate a refund back ESPECIALLY if no one responds to the email support like I'm seeing on the other threads.

**Comment by gandolfslayer** [comment]
AND WHERES YOUR REGISTERED AGENT AT BRO? if you don't respond ill come find them

**Comment by benhalpern** [comment]
We have unsuspended your account upon review, sorry about the inconvenience.
